forms/models/form.js

```diff
@@ -371,6 +382,22 @@ define(function (require) {
     /* eslint-disable no-unused-vars */ //stop eslint compaining about options and errorList not being used.
     setErrors: function(errorList, options){
       var elementsCollection = this.get('elements');
+      var subForms = this.getSubforms();
+
+      _.each(subForms, function(subForm, name){
+        var subFormErrorList = _.omit(errorList[name], 'errors');
+
+        _.each(subFormErrorList, function(errors, formName){
+          var forms = subForms[name].where({name: formName});
+          _.each(errors, function(fieldError, formIndex){
+            var form = subForms[name].at(formIndex);
+            if ( form ){
+              form.setErrors(fieldError);
+            }
+          });
+        });
+      });
+
```

Comment:
I'm curious about this. Is a triple-nested `_.each()` going to be better than a triple-nested `for` loop? I'm worried about all the new functions being allocated. Is `setErrors()` is being throttled or debounced or something elsewhere? Is the input always going to be of a small enough size that this is already optimal enough?